apps/jenkins/jenkins/ptlsbox/jenkins-azure-vm-agent.yaml

```diff
@@ -10,6 +10,68 @@ spec:
       JCasC:
         configScripts:
           azure-vm-agent: |
+            x-vmTemplate-values: &vm_template_values_anchor
+              - launcher: "ssh"
```

Comment:
this shouldn't be an array, just an object, see https://github.com/hmcts/cnp-flux-config/blob/master/apps/jenkins/jenkins/sbox-intsvc/jenkins-azure-vm-agent.yaml

I would just copy paste that and then adjust